src/Sulu/Bundle/AdminBundle/Resources/js/components/Pagination/Pagination.js

```diff
@@ -68,8 +112,33 @@ export default class Pagination extends React.PureComponent<Props> {
         }
     };
 
+    @action handleInputChange = (value: ?string) => {
+        if (value === undefined) {
+            this.currentInputValue = undefined;
+            return;
+        }
+
+        const page = parseInt(value);
+
+        if (!isNaN(page)) {
+            this.currentInputValue = page;
+        }
+    };
+
+    @action handleInputBlur = () => {
+        const {currentPage, onPageChange} = this.props;
+        const page = this.getPage();
```

Comment:
I think this is the only line where we need to find the page in the way the `getPage` function does it, right? I think `getPage` is not a good name, because it is not clear what page it returns (could also be the current value of the input). Why don't we inline the function in here? Then we wouldn't have to find a name, because I can't think of a good one :see_no_evil: